src/lib/regexHelper.js

```diff
@@ -1,3 +1,4 @@
+const nonDigitsRegexp = /(\D\s+)/g;
 const allSpacesRegexp = / /g;
 const nonDigitSlashRegexp = /\D\/$/;
 const nonDigitRegexp = /\D/g;
```

Comment:
What was the different between `nonDigitRegexp` and `nonDigitsRegexp`?

Reply:
with the first one you mentioned (`nonDigitRegexp`) it was not taking the `new-line` at the end of the given value 🤷‍♂️